db/migrate/20171027115228_add_remember_token.rb

```diff
@@ -0,0 +1,5 @@
+class AddRememberToken < ActiveRecord::Migration
+  def change
+    add_column :users, :remember_token, :string, :limit => 150, :index => true
```

Comment:
I think we could place the column in a better place: `after: :reset_password_sent_at`, what do you think?

Reply:
no strong opinion on that, i updated it

Reply:
The schema has not been updated :)

Reply:
the schema is not used in production anyways. and i ran the migration again and the result is the same in `schema.rb`

Reply:
If you had already ran it yes, schema won't change. If you rerun the migration you'll see the column at the right position in the schema.rb file. 

We do use it the first time Velum runs (we do a load schema).

Reply:
BTW just saw that the syntax of the hash is not unified, maybe that's the issue.

Reply:
i rolled back and then re-ran it of course. but still there was no change. the hash syntax i dont think is the issue, but i can try again